class4.html

```diff
@@ -236,7 +239,128 @@ <h2>Set the browser tab Title</h2>
         </section>
 
         <section>
-          <h2>Add the Nav Content</h2>
+          <h2>Assess the Layout</h2>
+
+          <p>
+            Focusing on the top portion of our website, we have a vertical
+            split. Ignoring the content for now, let's focus on building this
+            layout.
+          </p>
+        </section>
+
+        <section>
+          <h2>Getting Started</h2>
+
+          <ol>
+            <li>Create a generic container using a div.</li>
+            <li>Add a class so we can style it with CSS</li>
+          </ol>
+
+          <pre>
+            <code class="html" data-trim contenteditable>
+              &lt;body&gt;
```

Comment:
Not a big deal but I notice that [later code](https://github.com/SViccari/web-development-intro/pull/13/files#diff-ce81b4db6e311d26883d4c1bf3bf3134R712) don't have to use these HTML entities which is easier to read.

Reply:
Unfortunately, if I write `<body>`, Reveal will evaluate the HTML and doesn't show the actual code sample `<body>`. The [docs](https://github.com/hakimel/reveal.js/#code-syntax-highlighting) state "HTML will be escaped by default" so I'd love to be wrong and avoid writing  `&lt;` and `&gt` for `html, head, body` tags but so far, I haven't found another way to write these code samples.